Applications/AdminUi/apps/admin_ui/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   admin_api_sdk: ^1.0.0
   admin_api_types: ^1.0.0
-  collection: ^1.18.0
+  collection: any
```

Comment:
Why don't we pin the version?

Reply:
There is another package (afaik test) that depends on collection. As we cannot bump it and we dont care for the version any is the solution!